vocabularies/UI.xml

```diff
@@ -688,6 +694,15 @@ The trend is
         <Property Name="Measures" Type="Collection(Edm.PropertyPath)" Nullable="false">
           <Annotation Term="Core.Description" String="Measures of the chart, e.g. size and color in a bubble chart" />
         </Property>
+        <Property Name="DynamicMeasures" Type="Collection(Edm.AnnotationPath)" Nullable="false">
```

Comment:
I assume we do not want `DynamicDimensions`.

Reply:
Compute the year from a date, then group by year. 

Would that be a dynamic dimension?

Reply:
> Would that be a dynamic dimension?

Yes, but have we got a term that allows us to express such a thing? Also, grouping by such computed dimensions is discouraged for performance reasons.